dask/array/utils.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+from .core import Array
+from ..async import get_sync
+
+def eq(a, b, **kwargs):
```

Comment:
Perhaps rename to `assert_equivalent`, or something that indicates that this provides the assertions, not just checks equivalence.